content/docs/glossary/muxed-accounts.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Multiplexed Accounts
+order:
+---
+
+import { Alert } from "components/Alert";
+
+<Alert>
+
+  **Warning**: *This feature is experimental.*
+  Multiplexed accounts are not widely supported yet. Many wallets may not display a muxed account address and instead display the account ID address, ignoring the 64-bit ID. Adoption efforts are actively in progress.
+
+</Alert>
+
+
+A **multiplexed** (or **muxed**) **account** is an account that exists "on top of" a traditional Stellar account address. It combines the familiar `GABC...` address alongside a 64-bit integer ID in order to support differentiating multiple "virtual" accounts that share an underlying "real" account.
```

Comment:
The term 'muxed' is well defined to be 'multiplexed' even in regular dictionaries. Given that we're calling them muxed accounts in SDKs and resolved to use that name for them, can we call them only muxed accounts rather than use both names in the doc?

Reply:
That seems fine with me, though from a cursory glance it seemed like the definition of "muxed" usually uses "multiplex" to define itself (super helpful :roll_eyes:). Notice that I included the "full" version of it just once here and use "muxed" exclusively since that's consistent with our SDKs, etc. as you mentioned and I agree that that's how we should refer to it. I just thought it'd be nice to at least associate the two words together for familiarity. Maybe switching the order ("**muxed** (or *multiplexed*) **account** is...") would be better?

Ultimately, though, I didn't participate in the naming discussions y'all had, so I'll follow whatever the consensus is!

Reply:
Good point. I think lead with muxed, and move multiplexec into the brackets is the right call. Keeping the bolded text the thing we want people to call it +1.

Reply:
Done!